scripts/utils/migrate/migrate-nav-structure.js

```diff
@@ -401,4 +404,4 @@ const write = (file, contents) => {
 const shouldRemoveNode = (node) =>
   !node.path && (!node.pages || node.pages.length === 0);
 
-module.exports = migrateNavStructure;
+module.exports = { migrateNavStructure };
```

Comment:
It looks like we are still importing this as [the default](https://github.com/newrelic/docs-website/pull/729/files#diff-fb209cfbca6f9a5efb2360c1518d59e68c6974106513282a3317e1ab4d158edeR16). Can you revert this, or update the import to destructure this?

Reply:
oh yep. thanks for catching that!